src/main/dynamo/Beyonce.ts

```diff
@@ -280,7 +280,37 @@ export class Beyonce {
       Put: { TableName: this.table.tableName, Item: item }
     }))
 
-    await this.client.transactWrite({ TransactItems: encryptedItems }).promise()
+    await this.client
+      .transactWrite({ TransactItems: encryptedItems })
+      .promise()
+  }
+
+  /** Write multiple items into Dynamo.
+   */
+  async batchPut<T extends TaggedModel>(params: {
+    items: T[];
+  }): Promise<{ unprocessedItems: T[] }> {
+    const maybeEncryptedItems = await this.maybeEncryptItems(params.items)
+    const encryptedItems = maybeEncryptedItems.map(item => ({
+      PutRequest: {
+        Item: item
+      }
+    }))
+
+    const { UnprocessedItems } = await this.client
+      .batchWrite({
+        RequestItems: {
+          [this.table.tableName]: encryptedItems
```

Comment:
If we had a `const { tableName } = this.table` above, it'd DRY things up a bit as we do it ~3 times.